src/sptrun.f

```diff
@@ -13,58 +12,45 @@
-C> @param MAXWV    - INTEGER SPECTRAL TRUNCATION
-C> @param IDRTI    - INTEGER INPUT GRID IDENTIFIER
-C>                (IDRTI=4 FOR GAUSSIAN GRID,
-C>                 IDRTI=0 FOR EQUALLY-SPACED GRID INCLUDING POLES,
-C>                 IDRTI=256 FOR EQUALLY-SPACED GRID EXCLUDING POLES)
-C> @param IMAXI    - INTEGER EVEN NUMBER OF INPUT LONGITUDES.
-C> @param JMAXI    - INTEGER NUMBER OF INPUT LATITUDES.
-C> @param IDRTO    - INTEGER OUTPUT GRID IDENTIFIER
-C>                (IDRTO=4 FOR GAUSSIAN GRID,
-C>                 IDRTO=0 FOR EQUALLY-SPACED GRID INCLUDING POLES,
-C>                 IDRTO=256 FOR EQUALLY-SPACED GRID EXCLUDING POLES)
-C> @param IMAXO    - INTEGER EVEN NUMBER OF OUTPUT LONGITUDES.
-C> @param JMAXO    - INTEGER NUMBER OF OUTPUT LATITUDES.
-C> @param KMAX     - INTEGER NUMBER OF FIELDS TO TRANSFORM.
-C> @param IPRIME   - INTEGER INPUT LONGITUDE INDEX FOR THE PRIME MERIDIAN.
-C>                (DEFAULTS TO 1 IF IPRIME=0)
-C>                (OUTPUT LONGITUDE INDEX FOR PRIME MERIDIAN ASSUMED 1.)
-C> @param ISKIPI   - INTEGER SKIP NUMBER BETWEEN INPUT LONGITUDES
-C>                (DEFAULTS TO 1 IF ISKIPI=0)
-C> @param JSKIPI   - INTEGER SKIP NUMBER BETWEEN INPUT LATITUDES FROM SOUTH
-C>                (DEFAULTS TO -IMAXI IF JSKIPI=0)
-C> @param KSKIPI   - INTEGER SKIP NUMBER BETWEEN INPUT GRID FIELDS
-C>                (DEFAULTS TO IMAXI*JMAXI IF KSKIPI=0)
-C> @param ISKIPO   - INTEGER SKIP NUMBER BETWEEN OUTPUT LONGITUDES
-C>                (DEFAULTS TO 1 IF ISKIPO=0)
-C> @param JSKIPO   - INTEGER SKIP NUMBER BETWEEN OUTPUT LATITUDES FROM SOUTH
-C>                (DEFAULTS TO -IMAXO IF JSKIPO=0)
-C> @param KSKIPO   - INTEGER SKIP NUMBER BETWEEN OUTPUT GRID FIELDS
-C>                (DEFAULTS TO IMAXO*JMAXO IF KSKIPO=0)
-C> @param JCPU     - INTEGER NUMBER OF CPUS OVER WHICH TO MULTIPROCESS
-C>                (DEFAULTS TO ENVIRONMENT NCPUS IF JCPU=0)
-C> @param GRIDI    - REAL (*) INPUT GRID FIELDS
-C> @param GRIDO    - REAL (*) OUTPUT GRID FIELDS
-C>                (MAY OVERLAY INPUT FIELDS IF GRID SHAPE IS APPROPRIATE)
-C>
-C>    CALLED:
-C>   SPTRAN       PERFORM A SCALAR SPHERICAL TRANSFORM
-C>   NCPUS        GETS ENVIRONMENT NUMBER OF CPUS
-C>
-C> REMARKS: MINIMUM GRID DIMENSIONS FOR UNALIASED TRANSFORMS TO SPECTRAL:
-C>   DIMENSION                    LINEAR              QUADRATIC
-C>   -----------------------      ---------           -------------
-C>   IMAX                         2*MAXWV+2           3*MAXWV/2*2+2
-C>   JMAX (IDRT=4,IROMB=0)        1*MAXWV+1           3*MAXWV/2+1
-C>   JMAX (IDRT=4,IROMB=1)        2*MAXWV+1           5*MAXWV/2+1
-C>   JMAX (IDRT=0,IROMB=0)        2*MAXWV+3           3*MAXWV/2*2+3
-C>   JMAX (IDRT=0,IROMB=1)        4*MAXWV+3           5*MAXWV/2*2+3
-C>   JMAX (IDRT=256,IROMB=0)      2*MAXWV+1           3*MAXWV/2*2+1
-C>   JMAX (IDRT=256,IROMB=1)      4*MAXWV+1           5*MAXWV/2*2+1
+C> Remarks: Minimum grid dimensions for unaliased transforms to spectral:
```

Comment:
Move this final table above the list of params.

In our docs we always have:

brief

complete description

params

author/date